ksp_plugin_adapter/burn_editor.cs

```diff
@@ -56,29 +56,45 @@ class BurnEditor : ScalingRenderer {
     ComputeEngineCharacteristics();
   }
 
+  public enum Event {
+    None,
+    Changed,
+    Deleted,
+    Minimized,
+    Maximized,
+  }
+
   // Renders the |BurnEditor|.  Returns true if and only if the settings were
   // changed.
-  public bool Render(string header,
+  public Event Render(string header,
                      bool anomalous,
-                     double burn_final_time,
-                     Action delete) {
+                     double burn_final_time) {
     bool changed = false;
     previous_coast_duration_.max_value = burn_final_time - time_base;
     using (new UnityEngine.GUILayout.HorizontalScope()) {
-      UnityEngine.GUILayout.Label(header);
-      string frame_info = "";
-      if (!reference_frame_selector_.FrameParameters().Equals(
+      if (UnityEngine.GUILayout.Button(
+              minimized ? "+" : "-", GUILayoutWidth(1))) {
+        minimized = !minimized;
+        return minimized ? Event.Minimized : Event.Maximized;
+      }
+      UnityEngine.GUILayout.Label(minimized ? $"{header} ({Δv():0.000} m/s)"
+                                            : header);
+      string info = "";
+      if (!minimized &&
+          !reference_frame_selector_.FrameParameters().Equals(
               adapter_.plotting_frame_selector_.FrameParameters())) {
-        frame_info = "Manœuvre frame differs from plotting frame";
+        info = "Manœuvre frame differs from plotting frame";
       }
       UnityEngine.GUILayout.Label(
```

Comment:
The placement of this string next to the `Delete` button is weird, it seems to invite you to delete the manœuvre when the frames don't match.  I would do something like:
```
Manœuvre #1 <grey>frame differs from plotting frame</grey>
```
That is, left align the message and shorten it a bit.